source/features/deep-reblame.tsx

```diff
@@ -7,11 +7,16 @@ import delegate, {DelegateEvent} from 'delegate-it';
 import * as api from '../libs/api';
 import features from '../libs/features';
 import loadingIcon from '../libs/icon-loading';
-import {getRepoGQL, getReference, looseParseInt} from '../libs/utils';
+import {getRepoGQL, getReference, looseParseInt, getCleanPathname} from '../libs/utils';
 
-const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number): Promise<string | false> => {
+const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number, currentFilename: string): Promise<string | Error> => {
```

Comment:
This only returns `string`; errors aren't typed 

```suggestion
const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number, currentFilename: string): Promise<string> => {
```